ports/boneraiserminions/port.json

```diff
@@ -12,7 +12,7 @@
       "Fraxinus88"
     ],
     "desc": "Amass an army of deamonic undead to fight endless waves of heroes intent to end your boneraising ways! In this gothic-horror auto-battle roguelite, use the bones of slain heroes to raise your minions, collect powerful relics and spells, and unlock meta upgrades, all to survive King Gigald's crusade!\n\nNote: This is the older version 3.1 which works with PortMaster!",
-    "inst": "Purchase the game from https://store.steampowered.com/app/1944570/Boneraiser_Minions/\nRun steam console via Windows+R: Type: steam://open/console, download version 3.1 by typing: download_depot 1944570 1944571 6000190403203003495 in the console, when the download is complete, it will tell you the location of the folder,  copy the data.win, audiogroup1.dat, audiogroup2.dat and options.ini from the folder to /ports/boneraiserminions/gamedata",
+    "inst": "Purchase the game from https://store.steampowered.com/app/1944570/Boneraiser\\_Minions/\nRun steam console via Windows+R: Type: steam://open/console, download version 3.1 by typing: download\\_depot 1944570 1944571 6000190403203003495 in the console, when the download is complete, it will tell you the location of the folder,  copy the data.win, audiogroup1.dat, audiogroup2.dat and options.ini from the folder to /ports/boneraiserminions/gamedata",
```

Comment:
```suggestion
    "inst": "Purchase the game from https://store.steampowered.com/app/1944570/Boneraiser_Minions/\nRun steam console via Windows+R: Type: steam://open/console, download version 3.1 by typing: download_depot 1944570 1944571 6000190403203003495 in the console, when the download is complete, it will tell you the location of the folder,  copy the data.win, audiogroup1.dat, audiogroup2.dat and options.ini from the folder to /ports/boneraiserminions/gamedata",
    "inst_md": "Purchase the game from https://store.steampowered.com/app/1944570/Boneraiser\\_Minions/\nRun steam console via Windows+R: Type: steam://open/console, download version 3.1 by typing: download\\_depot 1944570 1944571 6000190403203003495 in the console, when the download is complete, it will tell you the location of the folder,  copy the data.win, audiogroup1.dat, audiogroup2.dat and options.ini from the folder to /ports/boneraiserminions/gamedata",
```